BinaryEncoding.md

```diff
@@ -240,6 +239,42 @@ This section must be preceded by a [Functions](#functions-section) section.
 | count | `varuint32` | count of entries to follow |
 | entries | `uint16*` | repeated indexes into the function table |
 
+### Function Names section
+
+ID: `function_names`
+
+| Field | Type | Description |
+| ----- |  ----- | ----- |
+| names | `utf8-C-string*` | sequence of null-terminated utf8-encoded strings |
```

Comment:
Do we need to mandate utf8 null terminated? Could these also be length+bytes like other strings, for consistency?